custom_components/tahoma/sensor.py

```diff
@@ -86,15 +104,29 @@ def update(self):
         if CORE_RELATIVE_HUMIDITY_STATE in self.tahoma_device.active_states:
             self.current_value = float(
                 "{:.2f}".format(
```

Comment:
I had a review on a PR in core asking to use f-strings. should we use them instead of `"".format`?

Reply:
`f"{self.tahoma_device.active_states.get(CORE_RELATIVE_HUMIDITY_STATE):.2f}"`

Reply:
Not sure, I have seen the comment indeed, but this is how I retrieved it from the original and other components. We could change all of those .format to fstrings in another refactor PR.

Reply:
fine by me.

Reply:
https://developers.home-assistant.io/docs/development_guidelines/#use-new-style-string-formatting

Reply:
```
self.current_value = f"{self.tahoma_device.active_states.get(CORE_RELATIVE_HUMIDITY_STATE):.2f}"
```

I am not sure if this is more readable in this case, so I will leave it for now.